server/webapp/WEB-INF/rails.new/webpack/single_page_apps/analytics.js

```diff
@@ -32,12 +32,12 @@
 
   const models = {};
 
-  function ensureModel(uid, pluginId, metric) {
+  function ensureModel(uid, pluginId, type, id) {
     let model = models[uid];
 
     if (!model) {
       model = models[uid] = new Frame(m.redraw);
-      model.url(Routes.dashboardAnalyticsPath({plugin_id: pluginId, metric})); // eslint-disable-line camelcase
+      model.url(Routes.showAnalyticsPath(pluginId, type, id)); // eslint-disable-line camelcase
```

Comment:
I am ok to leave it as is for now since its not been used yet. Even if we want to change it the message from the iFrame does not have the `metric_id` and `type` to construct the url.

Reply:
We do know the `type` for both the calls, but yes `metric_id` is a new addition and needs to be passed along to construct the url. 
Why do you say these are not used? I thought these calls were used for the drilldown calls

Reply:
The drilldown is still WIP and needs to change after this PR gets merged. I can talk to @marques-work and keep him posted bout these changes.

Reply:
ok